docs/processing/qaqc-criteria-FC.md

```diff
@@ -0,0 +1,63 @@
+
+        ??? note "Another tweak possible would be to increase the motion censoring threshold"
+            However, in our study we pre-registered that we will perform the analysis with three different motion censoring threshold.
+            As such, the analysis with high motion censoring threshold will be reported anyway.
+
+
+    2. Only a few sessions' FC distribution are shifted towards positive values, skewed, flat or bimodal: then apply the [exclusion criteria detailed below](#fc-distributions-1)
+
+## QC-FC distributions
+
+- [ ] Check that the QC-FC distribution<sup>[2]</sup> and the null distribution represented by the red dotted line are approximately equivalent by assessing the QC-FC% value corresponding to the distance between the observed and the null distributions.
+$QC-FC%  \geq 95%$ is considered indicative of negligible modulations in the BOLD signal correlation structure <sup>[2]</sup>.
+$QC-FC% \le 95%$ lower require revising the preprocessing of the data as follows:
+    - [ ] Include more motion regressors by switching motion regression strategy using the `--motion "full"` flag when running `funconn.py`.
+    - [ ] Increase the cut-off frequency of the low-pass filtering using the `--low-pass` flag of `funconn.py` if the QC-FC distribution still does not meet the 95% cutoff.
+    - [ ] If tweaking the preprocessing does not suffice, revise [the exclusion criteria by excluding additional sessions](#qc-fc-distributions-1)
+    
+## QC-FC versus Euclidean distance
+
+- [ ] Check that there is no significant correlation between QC-FC correlations and Euclidean distance separating the nodes<sup>[2]</sup>. 
+If the correlation is significant, it indicates that connectivity estimates are biased in a manner that is distance-dependent, with many long-distance correlations decreased by motion and many short-distance correlations increased by motion <sup>[4]</sup>.
+As such, motion correction needs to be revised as follows:
+    - [ ] Include more motion regressors by switching motion regression strategy using the `--motion "full"` flag when running `funconn.py`.
+    - [ ] If tweaking the preprocessing does not suffice, revise [the exclusion criteria by excluding additional sessions](#qc-fc-versus-euclidean-distance-1)
+
+# Exclusion criteria for denoised data
+
+## FC distributions
+
+- [ ] Exclude the sessions for which the FC distribution appears shifted towards positive values, skewed, flat, or bimodal.
+Those sessions's BOLD signal are sill contaminated with residual noise, despite the preprocessing being adequate for the acquisition protocol. A likely scenario is that the subject moved a lot during this particular session.
+
+!!! note "It is unlikely such sessions did not get excluded at earlier checkpoints."
```

Comment:
We should be more specific, what does this comment help with?

```suggestion
    !!! note "It is unlikely such sessions did not get excluded at earlier checkpoints."
```

Reply:
I'm actually not sure

Reply:
If we are excluding the session at this point, is it really necessary the note?